exp/views/study.py

```diff
@@ -556,12 +585,11 @@ def get_study_attachments(self, study, orderby, match):
         return sorted(attachments, key=lambda x: getattr(x, sort), reverse=True if '-' in orderby else False)
 
     # TODO move to celery task
-    def download_all_files(self):
+    def download_multiple_files(self, all, folder_name):
```

Comment:
I might add the study uuid to the filename of the zip just so it's easier for the researchers.